app/certificates/route.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  title() {
+    var stack = this.modelFor('stack');
+    return `${stack.get('handle')} Certificates - ${stack.get('organization.name')}`;
+  },
+
+  model() {
+    var stack = this.modelFor('stack');
+    return stack.get('certificates');
+  },
+
+  afterModel(model) {
+    return model.map((certificate) => certificate.get('vhosts'));
```

Comment:
This seems to return an array of promises, but that doesn't have any "magic" from Ember.  You would need to return `Ember.RSVP.all(model.map(....))`.